### ISSUE TITLE
`cargo test` generated 1 warning in windows10

### ISSUE THREAD

**Issue by Sysix** [oxc-project/oxc#8719]
Creating this issue because it looks like only a small number people are tracking windows behavior.
All tests are passing but I got a warning at the top:

```bash
PS C:\dev\oxc> cargo test
    Blocking waiting for file lock on build directory
   Compiling fixedbitset v0.5.7
warning: error finalizing incremental compilation session directory `\\?\C:\dev\oxc\target\debug\incremental\oxc_isolated_declarations-05ztp4a6x57ws\s-h3zbj7uzkj-1742427-working`: Zugriff verweigert (os error 5)

warning: `oxc_isolated_declarations` (lib) generated 1 warning
   Compiling petgraph v0.7.1
   Compiling oxc_cfg v0.48.0 (C:\dev\oxc\crates\oxc_cfg)
   Compiling oxc_semantic v0.48.0 (C:\dev\oxc\crates\oxc_semantic)
   Compiling oxc_mangler v0.48.0 (C:\dev\oxc\crates\oxc_mangler)
   .....
```

"Zugriff verweigert" are the german words for "Access denied".
Feel free do close this issue when it is not important for you :)

**Comment by Boshen** [comment]
Is this reproducible? Feels like a cargo bug.

**Comment by shulaoda** [comment]
It works fine on my windows. Could u try running `cargo clean` & `rustup override set 1.84.0`?

**Comment by Sysix** [comment]
> Could u try running cargo clean & rustup override set 1.84.0

This solved the problem. Thanks!
